eve/main.yml

```diff
@@ -30,3 +41,17 @@ stages:
         urls:
           - "*.iso"
           - "SHA256SUM"
+
+  lint:
+    worker:
+      type: kube_pod
```

Comment:
Why not go with a simpler `docker` type worker? Ditto for build step by the way

Reply:
For the build step: I believe we can't do docker-in-docker directly in a `docker` worker, but we want to maintain 1:1 between local usage and CI usage, so we build the same way in both (!= RING in that regard).

For the linting step: no argument against. I don't foresee the need for a pod, so I can switch it to docker. We need to discuss more about the lint/test separation.